driver.go

```diff
@@ -256,13 +258,29 @@ func getEnvDuration(env string, dValue time.Duration) time.Duration {
 		retDuration, err = time.ParseDuration(eDuration)
 		if err != nil {
 			logrus.Error(fmt.Sprintf("Error parsing drain timeout %s\n", err))
-			logrus.Info(fmt.Sprintf("Using default drain timeout %v\n", dValue))
-			retDuration = defaultLogsDrainTimeout
+			logrus.Info(fmt.Sprintf("Using default drain timeout %+v\n", dValue))
+			return defaultLogsDrainTimeout
 		}
 	}
 	return retDuration
 }
 
+func getEnvBool(env string, dValue bool) bool {
```

Comment:
whats this for?

Reply:
Parsing environment variable (string) to its boolean value